.github/workflows/periodic.yml

```
@@ -8,6 +8,30 @@ on:
 permissions: {}
 
 jobs:
+  internal-integration:
+    name: Internal Integration
+    if: github.repository_owner == 'submariner-io'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
+
+      - name: Update internal submariner-io/* dependencies to latest
+        run: |
+          go get github.com/submariner-io/admiral@devel
+          go get github.com/submariner-io/shipyard@devel
```

We can make this generic:

```suggestion
          for dep in $(awk '!/module/ && /github.com.submariner-io/ { print $1 }' go.mod); do go get ${dep}@devel; done
```